dotcom-rendering/src/client/userFeatures/userBenefitsApi.ts

```diff
@@ -0,0 +1,39 @@
+import { isObject } from '@guardian/libs';
+import {
+	getOptionsHeadersWithOkta,
+	type SignedInWithCookies,
+	type SignedInWithOkta,
+} from '../../lib/identity';
+import { fetchJson } from './fetchJson';
+import type { UserBenefits } from './user-features';
+
+type UserBenefitsResponse = {
+	benefits: string[];
+};
+export const syncDataFromUserBenefitsApi = async (
+	signedInAuthStatus: SignedInWithOkta | SignedInWithCookies,
+): Promise<UserBenefits> => {
+	const response = await fetchJson(
+		window.guardian.config.page.userBenefitsApiUrl ??
+			'/USER_BENEFIT_API_NOT_FOUND',
```

Comment:
Would it be clearer/more explicit to error in the case that the url isn't set in config? (Rather than making a request to a non-existent URL?)

Reply:
I did think that as well, but I stuck with the way it was written already. Happy to change it, what do you think?

Reply:
I went ahead and changed it.

Reply:
Ah great, yeah I think it feels like there'd be less scope for confusion when debugging an issue that way 👍